Gemfile

```diff
@@ -9,22 +9,24 @@ gem 'bootstrap-will_paginate', '0.0.10'
 gem 'bootstrap_form', '2.5.2'
 gem 'chartkick', '2.2.1' # Chart project_stats
 gem 'faker', '1.6.6'
-gem 'fastly-rails', '0.7.1' # Use Fastly CDN
+gem 'fastly-rails', git: 'https://github.com/dankohn/fastly-rails',
+                    ref: 'a909d09'
```

Comment:
Good point.   I've added a new issue, #558, to track enabling these new Rails 5 hardening mechanisms.